static/src/javascripts/projects/common/modules/discussion/loader.js

```diff
@@ -20,7 +20,7 @@ import {
     handle as upvoteHandle,
     closeTooltip as upvoteCloseTooltip,
 } from 'common/modules/discussion/upvote';
-import { getUserFromCookie, getUserFromApi } from 'common/modules/identity/api';
+import { getUserFromCookie, getUserFromApiOrOkta } from 'common/modules/identity/api';
```

Comment:
I don't think we're using this import anywhere!

```suggestion
import { getUserFromCookie } from 'common/modules/identity/api';
```